tempesta_fw/server.h

```diff
@@ -64,9 +65,15 @@ typedef struct {
 	atomic64_t		refcnt;
 	unsigned int		weight;
 	unsigned int		flags;
+	unsigned int		hm_flags;
```

Comment:
Correct me if I am wrong, but my understanding is that both atomic and non-atomic ops can be used on the same variable. There's only a handful of flags, and it's a 32-bit variable. So, perhaps, a separate `hm_flags` is unnecessary?

Reply:
In current implementation of `set_bit/clear_bit` functions for x86, in cases of constant bit index argument - per-byte instructions `orb` and `andb` are used. So if for bit index we use immediate value less then 32 - we will not go beyond the `int` boarders. But yes, this is not very safe approach.
Corrected.